src/Protocol/Protocol_1_13.cpp

```diff
@@ -173,7 +174,32 @@ void cProtocol_1_13::SendTabCompletionResults(const AStringVector & a_Results)
 
 void cProtocol_1_13::SendUpdateBlockEntity(cBlockEntity & a_BlockEntity)
 {
-	// TODO
+	ASSERT(m_State == 3);  // In game mode?
+	cPacketizer Pkt(*this, pktUpdateBlockEntity);
+	cFastNBTWriter Writer;
```

Comment:
I don't think Writer is being used

Reply:
You are right- this is a leftover from me trying out some things

Reply:
done